### ISSUE TITLE
"admin/release-twisted --help" contains typo

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1970]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1970|
|Type|defect|
|Created|2006-07-29 17:44:23Z|

It talks about "--command" but the argument is "--commands"

Attachments:

* [patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/f99/f9922496ec864c9b125d22f29bae753da6b52e17/d75b3b528276d7a9f30a04b8bccaf74e1d61f67a) (307 bytes) - added by radix on 2006-09-11 23:09:34Z - dfix it

<details><summary>Searchable metadata</summary>

```
trac-id__1970 1970
type__defect defect
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1154195063000000 1154195063000000
changetime__1158877885000000 1158877885000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_3"></a>|@jml set owner to @radix|
|-|-|

Looks good, please merge

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set status to `closed`|
|-|-|

(In [18213]) Apply patch from [[#1970](https://github.com/twisted/twisted/issues/1970)](https://github.com/twisted/twisted/issues/1970)

Author: radix
Reviewer: jml
Fixes #1970

Fix a typo that where "--command" was used instead of "--commands".

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
